File: laptop/C_Solution/단계별/브루트포스/1436.c
// Movie Director

#include <stdio.h>
#include <math.h>
#define True 1
#define False 0

int check(int num){
    for(int i = 0; i < 10; i++){
        if(num / (int)pow(10, i) % 1000 == 666){
            return True;
        }
    }
    return False;
}

int main(){
    int n,num=666,cnt=1;
    scanf("%d", &n);

    while(True){
        int flag = check(num);
        if(flag == True){
            if(cnt==n) break;
            else cnt++;
        }
        num++;
    }
    printf("%d", num);
    return 0;
}

File: laptop/Python_Solution/단계별/브루트포스/1436.py
# Movie Director

"""
n=int(input())
check=666
cnt=1
while 1:
    if "666" in str(check):
        if cnt==n:
            break
        else: cnt+=1
    check+=1
print(check)
"""

# 10 => 2666799

print([i for i in range(9**7)if "666"in str(i)][int(input())-1])